src/main/java/com/redhat/cloud/notifications/routers/ApplicationService.java

```diff
@@ -26,9 +26,9 @@
     ApplicationResources appResources;
 
     @GET
-    public List<Application> getApplications() {
+    public Uni<List<Application>> getApplications() {
         // Return configured with types?
-        return appResources.getApplications().collectItems().asList().await().indefinitely();
+        return appResources.getApplications().collectItems().asList();
```

Comment:
I think we can go even further and revert @burmanm workaround to return a `Multi<Application>`

Reply:
Ah yes, check: https://gist.github.com/burmanm/ce371a45b78b290e77763dd5c0dfbd62